cmd/fleet/main.go

```diff
@@ -860,3 +879,63 @@ func (f *FleetServer) Reload(ctx context.Context, cfg *config.Config) error {
 	}
 	return nil
 }
+
+func (f *FleetServer) initTracer(cfg config.Instrumentation) error {
+	// TODO(marclop) is there an APM Go agent environment variable that could
```

Comment:
There's two environment variables which may be relevant depending on what you want to do:
 - [`ELASTIC_APM_ACTIVE`](https://www.elastic.co/guide/en/apm/agent/go/current/configuration.html#config-active)
 - [`ELASTIC_APM_RECORDING`](https://www.elastic.co/guide/en/apm/agent/go/current/configuration.html#config-recording)

I guess we want `ELASTIC_APM_ACTIVE`? Not creating a tracer is also a valid choice though.

Reply:
I decided to remove that comment and to not handle any of these environment variables since the tracer constructor will already behave the way we'd like. 
Let me know if you'd rather not create a tracer object when `ELASTIC_APM_ACTIVE` is set to `false`, but the overhead here should be minimal.

Reply:
SGTM